x-pack/plugins/dashboard_enhanced/public/services/drilldowns/dashboard_to_dashboard_drilldown/drilldown.test.tsx

```diff
@@ -4,17 +4,335 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { DashboardToDashboardDrilldown } from './drilldown';
+import { UrlGeneratorContract } from '../../../../../../../src/plugins/share/public';
+import { savedObjectsServiceMock } from '../../../../../../../src/core/public/mocks';
+import { dataPluginMock } from '../../../../../../../src/plugins/data/public/mocks';
+import { ActionContext, Config } from './types';
+import {
+  Filter,
+  FilterStateStore,
+  Query,
+  RangeFilter,
+  TimeRange,
+} from '../../../../../../../src/plugins/data/common';
+import { esFilters } from '../../../../../../../src/plugins/data/public';
+
+// convenient to use real implementation here.
+import { createDirectAccessDashboardLinkGenerator } from '../../../../../../../src/plugins/dashboard/public/url_generator';
```

Comment:
I'm surprised this isn't throwing an error for invalid deep import.

Reply:
Tests are excluded from that rule